Components/Auth/SocialAuthButton.js

```diff
@@ -28,7 +28,12 @@ const SocialAuthButton = ({ platform }) => {
             },
         })
     }
-
+    
+    var loginText = "Log In With " + platform.friendlyName
```

Comment:
To make this a tad cleaner, we should change the value to:
``Login With ${platform.friendlyName}``